### PR TITLE
Feature/support for 3100 and 3500

### PR DESCRIPTION
**Pull request by DaveAtkinsonXMOS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # xCORE VocalFusion XVF3510 kit for Amazon AVS on a Raspberry Pi
 
-The XMOS **xCORE VocalFusion XVF3510 kit for Amazon AVS** provides far-field voice capture using the XMOS XVF3500 voice processor.
+The XMOS **xCORE VocalFusion XVF3510 kit for Amazon AVS** provides far-field voice capture using the XMOS XVF3510 voice processor. It also supports the XMOS **xCORE VocalFusion Stereo 4-Mic Kit for AVS** using the XMOS XVF3500 voice processor.
```
Comment (lucianomartin) [inline README.md:3]: We should the info about the XVF3100 here

File: README.md [hunk 2]
```diff
@@ -12,7 +12,8 @@ This repository provides a simple-to-use automated script to install the Amazon
 ## Prerequisites
 You will need:
 
-- **xCORE VocalFusion XVF3510 kit for Amazon AVS**: XK-VF3510-L71
+- Either **xCORE VocalFusion XVF3510 Kit for Amazon AVS**: XK-VF3510-L71
```
Comment (lucianomartin) [inline README.md:15]: We should the info about the XVF3100 here

File: README.md [hunk 3]
```diff
@@ -12,7 +12,8 @@ This repository provides a simple-to-use automated script to install the Amazon
...
+- or     **xCORE VocalFusion Stereo 4-Mic Kit for Amazon AVS**: XK-VF3500-L33-AVS
 - Raspberry Pi 3
```
Comment (lucianomartin) [inline README.md:17]: It support both RPi4 and RPi3 with Buster

**Review by lucianomartin — CHANGES_REQUESTED** [review]
Just few comments

**Comment by DaveAtkinsonXMOS** [comment]
This work will be committed into the vocalfusion-avs-setup repo.

**Pull request closed without merging**